test/text/vtt_text_parser_unit.js

```diff
@@ -585,6 +585,19 @@ describe('VttTextParser', () => {
         {periodStart: 0, segmentStart: 0, segmentEnd: 0});
   });
 
+  it('supports extra newlines in cue body', () => {
+    verifyHelper(
+        [
+          {startTime: 20, endTime: 40, payload: ''},
```

Comment:
Shouldn't the payload be `Test\n\nExtra line`?

Reply:
Both `[timestamp]\n\n[cue]` and `[timestamp]\n[cue]\n\n[morecue]` are VTT errors. I will split this into two tests.

Reply:
Split with f569aa3.